app/src/main/kotlin/xyz/block/augurref/api/FeeEstimateEndpoint.kt

```diff
@@ -57,18 +59,46 @@ fun Route.configureFeesEndpoint(mempoolCollector: MempoolCollector) {
 
   get("/fees") {
     logger.info("Received request for fee estimates")
-    val currentEstimate = mempoolCollector.getLatestFeeEstimate()
 
-    if (currentEstimate == null) {
-      logger.warn("No fee estimates available yet")
+    // Extract date param from query parameters
+    val dateParam = call.request.queryParameters["date"]
+
+    // Validate and parse the date if provided
+    logger.info("Fetching fee estimate for dateParam: $dateParam")
+    val date = try {
+      dateParam?.let { LocalDateTime.parse(it) }
+    } catch (e: DateTimeParseException) {
+      logger.warn("Invalid date format: $dateParam")
       call.respondText(
-        "No fee estimates available yet",
+        "Invalid date format. Use YYYY-MM-DDTHH:MM:SS",
+        status = HttpStatusCode.BadRequest,
+        contentType = ContentType.Text.Plain,
+      )
+      return@get
+    }
+
+    logger.info("Fetching fee estimate for date: $date")
```

Comment:
remove this line

Reply:
updated in [c176e20](https://github.com/block/bitcoin-augur-reference/pull/6/commits/c176e20119154ad83f0f9e57a65d31995cef5a68)